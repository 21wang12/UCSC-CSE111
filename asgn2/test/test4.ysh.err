yshell: mkdir: /this/mkdir/should/error: doesn't exist
yshell: make: /foo/bar: is a directory
