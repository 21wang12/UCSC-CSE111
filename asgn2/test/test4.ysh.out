./yshell build Apr 29 2022 19:04:34
% mkdir foo
% mkdir foo/bar
% mkdir foo/bar/baz
% make foo/bar/baz/file with this foo bar baz in the file
% mkdir this/mkdir/should/error/out
% make foo/bar
% # make foo/bar should fail because it is a directory
% lsr /
/:
     1       3  ./
     1       3  ../
     2       3  foo/
/foo:
     2       3  ./
     1       3  ../
     3       3  bar/
/foo/bar:
     3       3  ./
     2       3  ../
     4       3  baz/
/foo/bar/baz:
     4       3  ./
     3       3  ../
     5       8  file
% cd foo
% make file8 nine ten eleven
% cat file8
nineteneleven
% cd /
% lsr /
/:
     1       3  ./
     1       3  ../
     2       4  foo/
/foo:
     2       4  ./
     1       3  ../
     3       3  bar/
     6       3  file8
/foo/bar:
     3       3  ./
     2       4  ../
     4       3  baz/
/foo/bar/baz:
     4       3  ./
     3       3  ../
     5       8  file
% lsr foo/bar
/foo/bar:
     3       3  ./
     2       4  ../
     4       3  baz/
/foo/bar/baz:
     4       3  ./
     3       3  ../
     5       8  file
% rmr foo
% lsr /
/:
     1       2  ./
     1       2  ../
% # This tests decoding pathnames
% # $Id: test4.ysh,v 1.3 2022-04-22 10:41:38-07 - - $
% ^D
yshell: exit(1)
